packages/typespec-vscode/src/typespec-utils.ts

```diff
@@ -198,3 +199,114 @@ export async function installCompilerWithUi(
   }
   return result;
 }
+
+function getConfigEntriesFromEmitterOptions(
+  emitterOptions: Record<string, any>,
```

Comment:
it's better to pass in the whole config so that the caller doesnt need to worry about it too.